### ISSUE TITLE
New board based on your observations

### ISSUE THREAD

**Issue by eNBeWe** [paulvha/multichannel-gas#1]
Hi,

I read your comments at the original repository as well as your document here.
Thanks for all the docmentation you provided.

I built a new board, based on the original Grove board and your remarks. If you are interested, you can have a look [at my first blogpost on that.](https://blog.enbewe.de/posts/2018-05-19-improving-a-gas-sensor/)

I will publish the KiCad files and firmware files as soon as possible.

**Comment by paulvha** [comment]
Thanks for sharing Nils

Sounds interesting. Do you the schematic of the 6814 board from aldiexpress? What are the resistor values and where do they connect to ?

Regards,
Paul

Verzonden vanuit Mail<https://go.microsoft.com/fwlink/?LinkId=550986> voor Windows 10

________________________________
Van: Nis Wechselberg <notifications@github.com>
Verzonden: Sunday, May 20, 2018 1:50:20 PM
Aan: paulvha/multichannel-gas
CC: Subscribed
Onderwerp: [paulvha/multichannel-gas] New board based on your observations (#1)


Hi,

I read your comments at the original repository as well as your document here.
Thanks for all the docmentation you provided.

I built a new board, based on the original Grove board and your remarks. If you are interested, you can have a look at my first blogpost on that.<https://eur03.safelinks.protection.outlook.com/?url=https%3A%2F%2Fblog.enbewe.de%2Fposts%2F2018-05-19-improving-a-gas-sensor%2F&data=02%7C01%7C%7Cf8636d9215364345e14008d5be47dd8a%7C84df9e7fe9f640afb435aaaaaaaaaaaa%7C1%7C0%7C636624138219263850&sdata=j22%2B%2F0c%2BMtgDLjB1E7PQnm2q7agL%2B2F7x9IPTxOPPJM%3D&reserved=0>

I will publish the KiCad files and firmware files as soon as possible.

**Comment by eNBeWe** [comment]
I do not have an official schematic but from what I can see/probe it looks like it is just an implementation of the reference design from the datasheet. The resistor values are matching the figure on page 3.

Apart from that there only seem to be two bypass capacitors on the board.  

EDIT:
I think this should be a close approximation of the Aliexpress Board:
![mics6814](https://user-images.githubusercontent.com/3796429/40282739-f92888b4-5c73-11e8-9374-a6e7b49eb248.png)

I uploaded the first firmware (replicating the Grove sensor behaviour) at
[https://github.com/eNBeWe/MiCS6814-I2C-Firmware](https://github.com/eNBeWe/MiCS6814-I2C-Firmware)
and the KiCad design files at
[https://github.com/eNBeWe/MiCS6814-I2C-PCB](https://github.com/eNBeWe/MiCS6814-I2C-PCB)

It would be great if you could have a look and tell me what you think about that.

**Comment by paulvha** [comment]
Thanks Nis

Will be interesting to see what kind of results this provides. Keep me posted

Regards,
Paul




Verzonden vanuit Mail<https://go.microsoft.com/fwlink/?LinkId=550986> voor Windows 10

________________________________
Van: Nis Wechselberg <notifications@github.com>
Verzonden: Sunday, May 20, 2018 2:53:55 PM
Aan: paulvha/multichannel-gas
CC: paulvha; Comment
Onderwerp: Re: [paulvha/multichannel-gas] New board based on your observations (#1)


I do not have an official schematic but from what I can see/probe it looks like it is just an implementation of the reference design from the datasheet. The resistor values are matching the figure on page 3.

Apart from that there only seem to be two bypass capacitors on the board.

**Comment by strikerdlm** [comment]
I have been reviewing the documents and I find it very interesting. Great Job! I find the assumed calibration from the factory somehow obscure. Have you looked into the sensitivity and specificity of the sensor? I know you have some calculations to determine the R2, did you get the confidence intervals for these calculations? It would be nice to gather data on how these sensors behave in contrast to measurements taken from reliable industrial hygiene equipment. Great job

**Comment by paulvha** [comment]
hi

I have not done further investigations on this multichannel gas board. Not only do I question the assumed calibration, but also concluded a hardware design in the heating circuit. To my opinion the heater does not get to the right temperature due to lack of power.

couple of words about these MOS sensors

I have now worked on a number of sensors that mostly work according to the same principle, but have not been able to find matching results (or close) among them in results. They all change resistor values based on the air/pollution changes enough to detect that "something has changed to the worse or better". It is then easy to build a circuit that reacts on that. However the exact measurements differ a lot in the same environment.

When the resistor value changes, we still do not know which gas / pollution is causing that. So while there are curves for the different gases, e.g. Ammonia, Hydrogen, CO etc.  we do not know which one it is (or combination).

I have been working with SCD30 NDIR sensor. That is sensitive to the size of the CO2 (um). See below a chart.  So a change in measurement result is expected to most likely have been caused by Co2.  I have now ordered different NDIR device and want to whether I get similar results.

[cid:a8b0ee12-45e4-46d6-bf1a-79e9a459fc95]
________________________________
Van: strikerdlm <notifications@github.com>
Verzonden: donderdag 15 november 2018 22:49
Aan: paulvha/multichannel-gas
CC: paulvha; Comment
Onderwerp: Re: [paulvha/multichannel-gas] New board based on your observations (#1)


I have been reviewing the documents and I find it very interesting. Great Job! I find the assumed calibration from the factory somehow obscure. Have you looked into the sensitivity and specificity of the sensor? I know you have some calculations to determine the R2, did you get the confidence intervals for these calculations? It would be nice to gather data on how these sensors behave in contrast to measurements taken from reliable industrial hygiene equipment. Great job
